Source/ChocolateyGui.Common.Windows/Views/SettingsView.xaml

```diff
@@ -64,7 +64,8 @@
 
                     <TextBlock Grid.Column="0" Grid.Row="0" Style="{StaticResource BaseAltTextBlockStyle}" Text="{Binding Name}" TextWrapping="Wrap" />
                     <metro:ToggleSwitch Grid.Column="1" Grid.Row="0" VerticalAlignment="Center" Margin="4" IsOn="{Binding Enabled}" cal:Message.Attach="[Event Toggled] = [Action ChocolateyFeatureToggled($dataContext)]"/>
-                    <TextBlock Grid.Column="0" Grid.ColumnSpan="2" Grid.Row="1" Style="{StaticResource BodyTextBlockStyle}" Text="{Binding Description}" TextWrapping="Wrap" />
+                    <!-- Remove the following FlowDirection once we can translate Chocolatey feature descriptions -->
+                    <TextBlock Grid.Column="0" Grid.ColumnSpan="2" Grid.Row="1" Style="{StaticResource BodyTextBlockStyle}" Text="{Binding Description}" TextWrapping="Wrap" FlowDirection="LeftToRight" />

```

Comment:
Can you capture an issue to make sure that this doesn't get forgotten about?

Reply:
Done https://github.com/chocolatey/ChocolateyGUI/issues/912